connector/mqtt/vernemq/v2k-brigde/bin/entryPoint.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# set -ex
+
+DATA_BROKER="${DATA_BROKER_DNS}:${DATA_BROKER_PORT}"
+
+VERNE_WEBHOOK_KAFKA_HOSTS=${KAFKA_HOSTS:-"kafka-server:9092"}
+VERNE_WEBHOOK_DATA_BROKER_HOST=${DATA_BROKER:-"data-broker:80"}
+
+# check if kafka is up
+kafka_hosts=$(echo $VERNE_WEBHOOK_KAFKA_HOSTS | tr "," "\n")
+
+for address in $kafka_hosts
```

Comment:
You shouldn't wait for all kafka instances. The idea of the list of brokers is to have more options if some of them are down.